pandas/tests/generic/test_generic.py

```diff
@@ -948,3 +947,21 @@ def test_axis_classmethods(self, box):
             assert obj._get_axis_number(v) == box._get_axis_number(v)
             assert obj._get_axis_name(v) == box._get_axis_name(v)
             assert obj._get_block_manager_axis(v) == box._get_block_manager_axis(v)
+
```

Comment:
this needs to go in pandas/tests/frame/test_dtype with the other astype tests.

add similar for series (or does this not affect series?)

Reply:
Moved

Reply:
Series testing seems to be unnecessary as an error is thrown by another method prior to the astype method getting a change to throw one.